### ISSUE TITLE
Releases lack jsonnetfmt cmd

### ISSUE THREAD

**Issue by FraBle** [google/go-jsonnet#401]
The archives provided (`.tar.gz` & `.zip`) as GitHub releases lack the `jsonnetfmt` command under `./cmd`.

```bash
can't load package: package ./cmd/jsonnetfmt: cannot find package "." in:
	/private/tmp/go-jsonnet-20200413-91992-2xpfjb/go-jsonnet-0.15.0/src/github.com/google/go-jsonnet/cmd/jsonnetfmt
```

**Comment by FraBle** [comment]
I guess it will come with the next release...
When is the next release planned?

**Comment by ghostsquad** [comment]
I didn't even know jsonnetfmt was out! Yes, when is this included?

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
Just ran into this as well. Would be great to have a release.

**Comment by PaulRudin** [comment]
There's now a github action to automatically add both binaries (for various platforms) built from the go sources when a new release is made. 

The existing releases from the c++ sources don't include jsonnetfmt (I think it was added after the travis config for building the binaries). Is it still desirable to have binaries built from the c++ sources as well as those from the go sources? If so I can add something for jsonnetfmt the existing config.

**Comment by sbarzowski** [comment]
Eventually we want to drop C++ jsonnetfmt in favor of Go implementation, so having C++-jsonnetfmt binary uploaded is not critical. If people end up using Go implementation, because that's more convenient, it would actually be a good thing.

**Comment by PaulRudin** [comment]
If you want to encourage using the go binaries over the c++ ones, then there's an argument in favour of removing the automatically built c++ ones, at least once there's a new release including the go binaries.

**Comment by ghostsquad** [comment]
Which GitHub action is this? I'm interested in creating a release for my own project when a new tag is pushed

**Comment by sbarzowski** [comment]
You can see it here: https://github.com/google/go-jsonnet/pull/444/files

**Comment by FraBle** [comment]
It's included now since [v0.16.0](https://github.com/google/jsonnet/releases/tag/v0.16.0).
